readme.md

```diff
@@ -132,7 +132,7 @@ It is worth, to begin with taxrules as it is the fastest.
 e.g.
 
 ```
-*/5 * * * * cd [full path to magento directory]/shell && /usr/bin/flock -n /tmp/vsf_index.lock  /usr/bin/php vsf_tools.php --action reindex 
+*/5 * * * * cd [full path to magento directory]/shell && /usr/bin/flock -n /tmp/vsf_index.lock  /usr/bin/php vsf_tools.php -- --action reindex 
```

Comment:
Do we need the `-- --action` or can we just do `--action`?

I ask because for me both work. So is the version better which you propose here?